### PR TITLE
Team setup + hopefully fixed an error

### PR DESCRIPTION
**Pull request by Zeni44**

### CODE REVIEW

**Comment by Zeni44** [comment]
Keep in mind this kinda kills serverside teams

**Comment by Techbot121** [comment]
`./notagain/lua/notagain/pac_server/autorun/join_leave_messages.lua:58: ')' expected (to close '(' at line 53) near 'end'` :suspect:

File: notagain/lua/notagain/pac_server/autorun/join_leave_messages.lua [hunk 1]
```diff
@@ -50,8 +50,10 @@ if SERVER then
 		
 	end)
 
-	function GAMEMODE:PlayerConnect() end
-	function GAMEMODE:PlayerDisconnected() end
+	hook.Add("Initialize",tag,function()
+		function GAMEMODE:PlayerConnect() end
+		function GAMEMODE:PlayerDisconnected() end
+	end
```
Comment (Techbot121) [inline notagain/lua/notagain/pac_server/autorun/join_leave_messages.lua]: missing ) here

**Review by Techbot121 — CHANGES_REQUESTED** [review]

**Comment by Zeni44** [comment]
Tfw no testing

**Review by Techbot121 — APPROVED** [review]

**Pull request merged**